devsetup/scripts/gen-edpm-node.sh

```diff
@@ -271,8 +272,14 @@ echo GATEWAY=$GATEWAY >> $NETSCRIPT
 echo DNS1=$DNS >> $NETSCRIPT
 sed -i s/dhcp/none/g $NETSCRIPT
 sed -i /PERSISTENT_DHCLIENT/d $NETSCRIPT
+
+# Additional commands
+
 EOF
 
+touch $FIRSTBOOT_EXTRA
+cat "$FIRSTBOOT_EXTRA" >> ${OUTPUT_DIR}/${EDPM_COMPUTE_NAME}-firstboot.sh
```

Comment:
I don't get it, it is an empty file, when/where do we add content to `/tmp/edpm-firstboot-extra`?